lomap/graphgen.py

```diff
@@ -726,21 +760,22 @@ def max_dist_mol(mol):
                 max_dist = max_dist_mol(mol)
 
                 if max_dist < self.max_mol_size:
-                    
-                    mol = AllChem.RemoveHs(mol)
-        
-                    AllChem.Compute2DCoords(mol)
-                
                     fname = os.path.join(directory_name, self.dbase[id_mol].getName() + ".png")
-                    
-                    Draw.MolToFile(mol, fname, size=(100,100), fitimage=True, imageType='png' )
-                    
+                    #sliu 07/17 
```

Comment:
changelog not necessary -- except that you want to make sure you explain exactly what you did here and why in the notes on the PR. Why are you changing this?